blocks/text.ts

```diff
@@ -648,16 +651,15 @@ blocks['text_reverse'] = {
   },
 };
 
-/**
- * @mixin
- * @package
- * @readonly
- */
+/** Type of a block that has QUOTE_IMAGE_MIXIN */
+type QuoteImageMixinBlock = Block&QuoteImageMixin;
+interface QuoteImageMixin extends QuoteImageMixinType {}
+type QuoteImageMixinType = typeof QUOTE_IMAGE_MIXIN;
```

Comment:
OK, but can we at least remove "Mixin" from `QuoteImageMixinBlock`?  Because otherwise (for consistency with the naming convention established everywhere else) the other two should be `QuoteImageMixinMixin` and `QuiteImageMixinMixinType`.